README.md

```diff
@@ -72,6 +72,26 @@ jobs:
       - uses: hynek/build-and-inspect-python-package@v2
 ```
 
+To also upload to PyPI:
+
+```yaml
+jobs:
+  check-publish-package:
+    name: Build, inspect, and upload our package to PyPI.
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: hynek/build-and-inspect-python-package@v2
```

Comment:
@jamesbraza doing this in the job with `id-token: write` is insecure and must be split: #151.